webpack/assets/javascripts/foreman_outdatedbrowser.js

```diff
@@ -0,0 +1,134 @@
+import '../styles/outdatedbrowser.scss';
+
+/*!--------------------------------------------------------------------
+JAVASCRIPT "Outdated Browser"
+Version:    1.1.2 - 2015
+author:     Burocratik
+website:    http://www.burocratik.com
+* @preserve
+-----------------------------------------------------------------------*/
```

Comment:
For the record this is MIT license: https://github.com/burocratik/outdated-browser/blob/develop/LICENSE

CHECK